main.js

```diff
@@ -14,7 +16,26 @@ registerPackageProtocol(cacheDir)
 
 const remixIdeUrl = 'package://6fd22d6fe5549ad4c4d8fd3ca0b7816b.mod'
 
-console.log('running', version)
+async function warnLatestVersion (current) {
+  try {
+    const res = await fetch('https://api.github.com/repos/ethereum/remix-desktop/releases/latest')
```

Comment:
I am not sure if `remix-desktop` tag should be checked as this implementation is for `remixd`

Reply:
here we are checking for new update of remix-desktop, not remixd

Reply:
ok